docs/content/integrations/bigquery.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "BigQuery + Dagster"
+description: Store your Dagster assets in BigQuery
+---
+
+# BigQuery + Dagster
+
+Using Dagster's [software-defined assets](/concepts/assets/software-defined-assets) and BigQuery I/O manager, you can easily interact with BigQuery tables alongside other Dagster assets.
+
+Managing your BigQuery tables with Dagster enables you to:
+
+- Use Python to analyze your data stored in BigQuery without writing custom SQL queries to fetch the data
+- Visualize the data dependencies between tables
+- Selectively update the contents of your tables
+- Integrate your Snowflake tables with other tools in your data stack
+
+---
+
+## BigQuery and Dagster tutorial
+
+In this tutorial, we'll learn how to store and load Dagster's [software-defined asset](/concepts/assets/software-defined-assets) in BigQuery. [Click here to get started](/integrations/bigquery/using-bigquery-with-dagster).
```

Comment:
```suggestion
In this tutorial, you'll learn how to store and load Dagster's [software-defined asset](/concepts/assets/software-defined-assets) in BigQuery. [Click here to get started](/integrations/bigquery/using-bigquery-with-dagster).
```

This might be a nit, but I think `you` makes more sense here. We as the teachers aren't learning anything we don't already know, but the reader is (hopefully!). If it was instead something like `we'll walk you through` or `we'll show you how to do [thing]`, that would also be fine.